test-utils/src/dev/falcon.rs

```diff
@@ -0,0 +1,13 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Mechanisms to launch and control propolis VMs via falcon
```

Comment:
Nit: Comment could probably be updated to indicate "this is an example falcon test"?

Reply:
Done.